treeherder/perf/models.py

```diff
@@ -100,6 +101,30 @@ def has_performance_data(self):
             signature_id=self.id,
         ).exists()
 
+    def has_data_with_historical_value(self):
```

Comment:
This starts to look like a big method & isn't easy to follow. Let's try & keep it under 10 lines of code.